observability/sops/alerts/rhoc_connectors_availability.adoc

```diff
@@ -0,0 +1,237 @@
+// begin header
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+:numbered:
+:toc: macro
+:toc-title: pass:[<b>Table of Contents</b>]
+// end header
+= RHOC Connectors Availability / SLO Error budget burn
+
+toc::[]
+
+== Description
+
+The purpose of this SOP is to outline the process of resolving the alerts `RHOCConnectorsAvailability*`. The rules for this alert operate within an OSD cluster. There are four error budget burn alerts that are used to protect the RHOC Connectors availability SLO.
+
+* slo_connector_availability_failure_rate:ratio_rate1h
+* slo_connector_availability_failure_rate:ratio_rate6h
+* slo_connector_availability_failure_rate:ratio_rate24h
+* slo_connector_availability_failure_rate:ratio_rate3d
+
+The source of the alert is the
+`cos_fleetshard_sync_connector_state_count_total` metric,
+and fires when it's error rate is high over a short window of time and a
+longer one, indicating that the error budget is being burned at an unsafe rate.
+
+The state of a connector determines the availability. A connector have 5 states:
+
+- CONNECTOR_STATE_READY
+- CONNECTOR_STATE_FAILED
+- CONNECTOR_STATE_DELETED
+- CONNECTOR_STATE_STOPPED
+- CONNECTOR_STATE_IN_PROCESS
+
+These vales are recorded in the label `cos_connector_state` which is part of `cos_fleetshard_sync_connector_state_count_total` metric.
+The error rate is calculated using  CONNECTOR_STATE_READY and CONNECTOR_STATE_FAILED. In order for this alert to trigger,
+the failing connector must reach CONNECTOR_STATE_READY before failing meaning the connector was able to run in the past.
+
+[NOTE]
+`cos_fleetshard_sync_connector_state_count_total` is reported by the cos-fleetshard-sync. Problems with the cos-fleetshard-sync could
+greatly affect the accuracy of this metric. Restarting the cos-fleetshard-sync pod will reset the `cos_fleetshard_sync_connector_state_count_total`
+counter and possibly tamper with the rate if the period of time used to calculate includes the restart.
+
+More info on this concept of multi-window multi-burn-rate alerts can be found
+https://sre.google/workbook/alerting-on-slos/#6-multiwindow-multi-burn-rate-alerts[here].
+
+More info on the SLO related to these alerts can be found
+https://docs.google.com/document/d/1GS8SROrDSZqWy6DD03TjJd0BM6OyLipv_BxAVb32XYk/edit#[here].
```

Comment:
Is this the right document for the connectors or does it share the doc with Kafka?

Reply:
Copy/Paste left over